jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/SparkWithDataFrame.scala

```diff
@@ -14,14 +14,13 @@
  limitations under the License.
  */
 
-package ml.dmlc.xgboost4j.scala.example.spark
```

Comment:
I moved this file from xgboost4j-example directory to xgboost4j-spark directory, which is more specific to spark issues.